README.md

```diff
@@ -1,6 +1,218 @@
+- [Gluster-Kubernetes Deploy Script](https://github.com/copejon/gk-cluster-deploy): to deploy GCP K8s cluster with Gluster-Kubernetes Object Storage and CNS Object Broker
+- Kubernetes-Incubator [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog): to provide Service-Catalog helm chart
+- Kubernetes [Helm](https://github.com/kubernetes/helm#install): to install Service-Catalog chart
+
+### Assumptions
+- Gk-cluster-deploy was written and tested on Fedora 25 and RHEL 7.
+- A [Google Cloud Platform](https://cloud.google.com/compute/) account.
+
+### Topology
+There are two primary systems that make up this demonstration.  They are the Broker
+and its colocated CNS Object store.  They fill the role of our micro-service provider.
+A second system will be the locally running Kubernetes cluster on which with Service-Catalog is deployed.  This cluster will be our micro-server client.  Please refer to the [command flow diagram](./docs/Service-Catalog-Published.png) for a more in depth look at these systems.
+
+## Setup
+### Step 0: Preparing environment
+- Clone [Kubernetes](https://github.com/kubernetes)
+- Clone [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog)
+- Install [Google SDK](https://cloud.google.com/sdk/) and add `gcloud` to `PATH`.  You must have an active Google Cloud Platform account.
+- Configure `gcloud` user, zone, and project.  These are detected by the deployment script and required for setup.
+
+
+`# gcloud config set account <user account>`
+
+`# gcloud config set project <project>`
+
+`# gcloud config set compute/zone <zone>`
+
+
+### Step 1: Deploy Gluster-Kubernetes cluster in Google Compute Engine (GCE)
+This step sets up the **External Service Provider** portion of our topology (see [diagram](./docs/Service-Catalog-Published.png)).
+
+To kick off deployment, run `gk-cluster-deploy/deploy/cluster/gk-up.sh`.  The script has a number of configurable
+variables relative to GCE Instance settings.  These can be defined inline with `gk-up.sh` or as environment variables.  To run the script as fire and forget, pass `-y` to skip configuration review.
+
+Runtime takes around 5 to 10 minutes.  Go get some coffee.
+
+Alternatively, you can manually deploy the GCE portion by following [these instuctions](//TODO).  // TODO
+
+When deployment completes, commands to ssh into the master node and the URL of the CNS Object Broker will be output. **Note the URL and PORT.**
+
+### Step 2: Deploy the Service-Catalog
+This step sets up the **K8s Cluster** portion of our topology (see [diagram](./docs/Service-Catalog-Published.png)).
+
+*In a separate terminal:*
+1. Change directories to the `service-catalog` repository.
+
+2. Follow the [Service-Catalog Installation instructions](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/introduction.md#installation).  Once the Service-Catalog is deployed, return here.
```

Comment:
Are these instructions accurate?

Reply:
Yes, a lot has changed.  At this point, our own documentation is inaccurate.  That link will take users to a page that has k8s version specific installation.

Reply:
Do you mean for debugging the storage component?